src/test/java/com/revature/personalfinance/service/AuthenticatorTests.java

```diff
@@ -0,0 +1,80 @@
+package com.revature.personalfinance.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class AuthenticatorTests {
+
+	// We will need more accurate JWTs that we can manipulate before being able to complete the testing
+	String validJWT = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9."
+			+ "eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ."
+			+ "N6_Glh0-bHSxjiZOgmIY6By_sog63l7bHRlxbExYPY8";
+	String invalidJWT = "eyJhbGci0iJTUzI1NiIsInR5cCI6IkpXVCJ9."
+			+ "eyJzdWIi0iIxMjM0NTY30DkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaXNTb2NpYWwiOnRydWV9."
+			+ "4pcPyMD09olPSyXnrXcjTwXyr4BsezdI1AVTmud2fU4";
+
+	/**
+	 * Method for testing isAuthentic() with a valid JWT
+	 */
+	@Test
+	public void testIsAuthenticSuccess() {
+		boolean result = Authenticator.isAuthentic(validJWT);
+		System.out.println("Authentic JWT case: " + validJWT);
```

Comment:
Should not commit sysout statements in the code base. If necessary to have this information it should be captured using loggers.

Reply:
All sysout statements will be removed.